.github/workflows/ci.yml

```diff
@@ -370,3 +370,34 @@ jobs:
           name: ${{ matrix.name }}_report
           path: |
             /tmp/report/
+
+  publish-modern-bpf-tests-artifacts-centos7:
+    runs-on: ubuntu-22.04 # maybe we have to decrease this one otherwise we cannot download kernel headers
+    steps:
+
+      - name: Install deps ⛓️
+        run: |
+          sudo apt update
+          sudo apt install -y --no-install-recommends ca-certificates cmake build-essential clang-14 git pkg-config autoconf automake libtool libelf-dev
+
+      - name: Checkout Libs ⤵️
+        uses: Andreagit97/checkout@v3
+        with:
+          fetch-depth: 0
+
+# Build the modern probe skeleton on Ubuntu 22.04 machine
+      - name: Build modern BPF skeleton 🏗️
+        run: |
+          mkdir -p skeleton-build
+          cd skeleton-build && cmake -DUSE_BUNDLED_DEPS=ON -DBUILD_LIBSCAP_MODERN_BPF=ON -DBUILD_LIBSCAP_GVISOR=OFF ../
+          make ProbeSkeleton
+
+      - name: Build the artifact from the docker image 🏗️
+        run: |
+          DOCKER_BUILDKIT=1 docker build -f ./docker/centos7-builder.Dockerfile --output type=local,dest=export-artifacts --build-arg CMAKE_OPTIONS="-DUSE_BUNDLED_DEPS=On -DBUILD_LIBSCAP_GVISOR=Off -DBUILD_BPF=True -DBUILD_LIBSCAP_MODERN_BPF=On -DCREATE_TEST_TARGETS=Off -DMODERN_BPF_SKEL_DIR=/libs/skeleton-build/skel_dir" .
```

Comment:
More a follow-up: I think it might be worth to have a follow-up using `docker buildx` to build the container on multiple platforms (have not yet tried this for falco libs).

Reply:
removed, same for all the other comments about ci or dockerfile :)